src/components/countries-dropdown/styles.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '~styles/settings';
+
+.expandIcon {
+  margin-left: 8px;
+}
+
+.countryButton {
+  color: $color-orange;
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+
+  svg {
+    fill: $color-orange;
+  }
+}
+
+.countryButtonActive {
+  svg {
+    transform: rotate(180deg);
+  }
+}
+
+.countriesContainer {
+  height: 150px;
+  width: 100%;
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 2;
+  background-color: $color-white;
+  border-top: 1px solid $color-black;
+  border-bottom: 1px solid $color-black;
+  display: flex;
+}
+
+.shape {
+  height: 100px;
+}
+
+.imageLink {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  height: 100%;
+  justify-content: center;
+  position: relative;
+
+  &:hover .shape__no-hover {
+    opacity: 0;
+    pointer-events: all;
+    transform: scale3d(0.8, 0.8, 1);
+  }
+
+  &:hover .shape__hover {
+    opacity: 1;
+    transform: scale3d(1, 1, 1);
+  }
+
+  .shape__hover {
+    position: absolute;
+    opacity: 0;
+    pointer-events: none;
```

Comment:
there's some sort of regression here, the hover would only trigger when on top of the active image. Now it's triggered even when on top of the white background

Reply:
fixed